appyters/Gene_Set_Library_Synopsis/README.md

```diff
@@ -1,2 +1 @@
-# Gene Set Library Synopsis
```

Comment:
Please restore the heading of the README

Reply:
I didn't mean to leave this off - thank you for pointing it out!